donut/modules/core/helpers.py

```diff
@@ -129,7 +129,7 @@ def get_group_list_of_member(user_id):
         result: All the groups that an user_id is a part of
     """
     query = """SELECT group_id, group_name, control
-    FROM group_members NATURAL JOIN groups NATURAL JOIN members
+    FROM positions NATURAL JOIN groups NATURAL JOIN position_holders
```

Comment:
Doesn't this need to take `positions_relations` into account?

Reply:
yup